pcsx2/GS/Renderers/HW/GSTextureCache.cpp

```diff
@@ -27,7 +27,7 @@
 
 #define XXH_STATIC_LINKING_ONLY 1
 #define XXH_INLINE_ALL 1
-#include "xxhash.h"
+#include "3rdparty/include/xxhash.h"
```

Comment:
Why was this changed?

Reply:
Because lz4 has its own `xxhash.h` header file and it was conflicting. Would you have a suggestion for a different way to prevent the header file conflict?

Reply:
How many files does lz4 actually need to compile? if it's not that many, you could extract the necessary files and just upload a custom setup as part of the PR instead of submoduling it, then you can link to our already included xxhash.

Reply:
You can just include the lz4.c/h we use on PSP/PS2, it's heavily stripped down and has no other dependency, just the c and h files.
https://github.com/ps2homebrew/Open-PS2-Loader/tree/master/modules/isofs